plugins/filter/src/src.pro

```diff
@@ -1,16 +1,12 @@
 include(../../plugin.pri)
 TARGET = voicecall-filter-plugin
 
-PKGCONFIG += mlite5
-
 DEFINES += PLUGIN_NAME=\\\"filter-plugin\\\"
 
-HEADERS += \
-    filterplugin.h \
-    filter.h \
-    filterlist.h
+INCLUDEPATH += $$PWD/../lib
+
+LIBS += -L$$PWD/../lib -lvoicecall-filter-plugin
```

Comment:
Library name not updated here to skip -plugin. Doesn't compile.

Reply:
Oops, sorry. Fixed.